### ISSUE TITLE
[Bug]: milvus服务关掉重启后search失败

### ISSUE THREAD

**Issue by chuzhifeng** [milvus-io/milvus#17563]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: v2.0.2
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):pymilvus 2.0.0
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

当服务重启后collection.search会卡住不work，插入新数据却可以

### Milvus Log

2022-06-15T08:52:22.896091756Z [2022/06/15 08:52:22.895 +00:00] [DEBUG] [impl.go:353] ["HasCollection received"] [traceID=21e7bd02f192dd28] [role=proxy] [db=] [collection=banban_image_machine_database]
2022-06-15T08:52:22.896296695Z [2022/06/15 08:52:22.896 +00:00] [DEBUG] [id.go:140] ["IDAllocator pickCanDoFunc"] [need=1] [total=199917] [remainReqCnt=0]
2022-06-15T08:52:22.896306488Z [2022/06/15 08:52:22.896 +00:00] [DEBUG] [impl.go:384] ["HasCollection enqueued"] [traceID=21e7bd02f192dd28] [role=proxy] [MsgID=433922062977335380] [BeginTS=433922544208183298] [EndTS=433922544208183298] [db=] [collection=banban_image_machine_database]
2022-06-15T08:52:22.896488888Z [2022/06/15 08:52:22.896 +00:00] [DEBUG] [root_coord.go:1354] [HasCollection] [role=rootcoord] ["collection name"=banban_image_machine_database] [msgID=433922062977335380]
2022-06-15T08:52:22.896568230Z [2022/06/15 08:52:22.896 +00:00] [DEBUG] [root_coord.go:1373] ["HasCollection success"] [role=rootcoord] ["collection name"=banban_image_machine_database] [msgID=433922062977335380]
2022-06-15T08:52:22.896678449Z [2022/06/15 08:52:22.896 +00:00] [DEBUG] [impl.go:414] ["HasCollection done"] [traceID=21e7bd02f192dd28] [role=proxy] [MsgID=433922062977335380] [BeginTS=433922544208183298] [EndTS=433922544208183298] [db=] [collection=banban_image_machine_database]
2022-06-15T08:52:22.898980155Z [2022/06/15 08:52:22.898 +00:00] [DEBUG] [impl.go:632] ["DescribeCollection received"] [traceID=772d91738fb62507] [role=proxy] [db=] [collection=banban_image_machine_database]
2022-06-15T08:52:22.899175589Z [2022/06/15 08:52:22.899 +00:00] [DEBUG] [id.go:140] ["IDAllocator pickCanDoFunc"] [need=1] [total=199916] [remainReqCnt=0]
2022-06-15T08:52:22.899186181Z [2022/06/15 08:52:22.899 +00:00] [DEBUG] [impl.go:656] ["DescribeCollection enqueued"] [traceID=772d91738fb62507] [role=proxy] [MsgID=433922062977335381] [BeginTS=433922544208183299] [EndTS=433922544208183299] [db=] [collection=banban_image_machine_database]
2022-06-15T08:52:22.899346805Z [2022/06/15 08:52:22.899 +00:00] [DEBUG] [root_coord.go:1394] [DescribeCollection] [role=rootcoord] ["collection name"=banban_image_machine_database] [msgID=433922062977335381]
2022-06-15T08:52:22.899425191Z [2022/06/15 08:52:22.899 +00:00] [DEBUG] [root_coord.go:1412] ["DescribeCollection success"] [role=rootcoord] ["collection name"=banban_image_machine_database] [msgID=433922062977335381]
2022-06-15T08:52:22.899587726Z [2022/06/15 08:52:22.899 +00:00] [DEBUG] [impl.go:689] ["DescribeCollection done"] [traceID=772d91738fb62507] [role=proxy] [MsgID=433922062977335381] [BeginTS=433922544208183299] [EndTS=433922544208183299] [db=] [collection=banban_image_machine_database]
2022-06-15T08:52:22.902016219Z [2022/06/15 08:52:22.901 +00:00] [DEBUG] [impl.go:448] ["LoadCollection received"] [traceID=551166d306d8cb6b] [role=proxy] [db=] [collection=banban_image_machine_database]
2022-06-15T08:52:22.902246140Z [2022/06/15 08:52:22.902 +00:00] [DEBUG] [id.go:140] ["IDAllocator pickCanDoFunc"] [need=1] [total=199915] [remainReqCnt=0]
2022-06-15T08:52:22.902257882Z [2022/06/15 08:52:22.902 +00:00] [DEBUG] [impl.go:470] ["LoadCollection enqueued"] [traceID=551166d306d8cb6b] [role=proxy] [MsgID=433922062977335382] [BeginTS=433922544208183300] [EndTS=433922544208183300] [db=] [collection=banban_image_machine_database]
2022-06-15T08:52:22.902284889Z [2022/06/15 08:52:22.902 +00:00] [DEBUG] [task.go:4253] ["loadCollectionTask PreExecute"] [role=proxy] [msgID=433922062977335382]
2022-06-15T08:52:22.902302607Z [2022/06/15 08:52:22.902 +00:00] [DEBUG] [task.go:4267] ["loadCollectionTask Execute"] [role=proxy] [msgID=433922062977335382]
2022-06-15T08:52:22.902360066Z [2022/06/15 08:52:22.902 +00:00] [DEBUG] [task.go:4289] ["send LoadCollectionRequest to query coordinator"] [role=proxy] [msgID=433922062977335382] [collectionID=433719451781431297] [schema="name:\"banban_image_machine_database\" description:\"Image hash search online collection\" fields:<fieldID:100 name:\"id\" is_primary_key:true data_type:Int64 autoID:true > fields:<fieldID:101 name:\"hash_vector\" data_type:BinaryVector type_params:<key:\"dim\" value:\"256\" > > "]
2022-06-15T08:52:22.902562235Z [2022/06/15 08:52:22.902 +00:00] [DEBUG] [impl.go:166] ["loadCollectionRequest received"] [role=querycoord] [collectionID=433719451781431297] [msgID=433922062977335382]
2022-06-15T08:52:22.902594708Z [2022/06/15 08:52:22.902 +00:00] [DEBUG] [impl.go:187] ["collection has already been loaded, return load success directly"] [role=querycoord] [collectionID=433719451781431297] [msgID=433922062977335382]
2022-06-15T08:52:22.902683598Z [2022/06/15 08:52:22.902 +00:00] [DEBUG] [task.go:4300] ["loadCollectionTask PostExecute"] [role=proxy] [msgID=433922062977335382]
2022-06-15T08:52:22.902695962Z [2022/06/15 08:52:22.902 +00:00] [DEBUG] [impl.go:500] ["LoadCollection done"] [traceID=551166d306d8cb6b] [role=proxy] [MsgID=433922062977335382] [BeginTS=433922544208183300] [EndTS=433922544208183300] [db=] [collection=banban_image_machine_database]
2022-06-15T08:52:22.904764438Z [2022/06/15 08:52:22.904 +00:00] [DEBUG] [impl.go:821] ["ShowCollections received"] [role=proxy] [DbName=] [TimeStamp=0] [ShowType=InMemory] [CollectionNames="[banban_image_machine_database]"]
2022-06-15T08:52:22.904946920Z [2022/06/15 08:52:22.904 +00:00] [DEBUG] [id.go:140] ["IDAllocator pickCanDoFunc"] [need=1] [total=199914] [remainReqCnt=0]
2022-06-15T08:52:22.904978785Z [2022/06/15 08:52:22.904 +00:00] [DEBUG] [impl.go:849] ["ShowCollections enqueued"] [role=proxy] [MsgID=433922062977335383] [DbName=] [TimeStamp=0] [ShowType=InMemory] [CollectionNames="[banban_image_machine_database]"]
2022-06-15T08:52:22.905130008Z [2022/06/15 08:52:22.905 +00:00] [DEBUG] [root_coord.go:1431] [ShowCollections] [role=rootcoord] [dbname=] [msgID=433922062977335383]
2022-06-15T08:52:22.905188360Z [2022/06/15 08:52:22.905 +00:00] [DEBUG] [root_coord.go:1449] ["ShowCollections success"] [role=rootcoord] [dbname=] ["num of collections"=1] [msgID=433922062977335383]
2022-06-15T08:52:22.905440594Z [2022/06/15 08:52:22.905 +00:00] [DEBUG] [impl.go:93] ["show collection start"] [role=querycoord] [collectionIDs="[433719451781431297]"] [msgID=433922062977335383]
2022-06-15T08:52:22.905471720Z [2022/06/15 08:52:22.905 +00:00] [DEBUG] [impl.go:148] ["show collection end"] [role=querycoord] [collections="[433719451781431297]"] [msgID=433922062977335383] [inMemoryPercentage="[100]"]
2022-06-15T08:52:22.905609389Z [2022/06/15 08:52:22.905 +00:00] [DEBUG] [impl.go:880] ["ShowCollections Done"] [role=proxy] [MsgID=433922062977335383] [DbName=] [TimeStamp=0] [ShowType=InMemory] [CollectionNames="[banban_image_machine_database]"] [result="status:<> collection_names:\"banban_image_machine_database\" collection_ids:433719451781431297 created_timestamps:433719451781431299 created_utc_timestamps:1654508406759 inMemory_percentages:100 "]
2022-06-15T08:52:22.982612810Z [2022/06/15 08:52:22.982 +00:00] [DEBUG] [impl.go:353] ["HasCollection received"] [traceID=70a66fa9d5ef1268] [role=proxy] [db=] [collection=banban_image_machine_database]
2022-06-15T08:52:22.982826424Z [2022/06/15 08:52:22.982 +00:00] [DEBUG] [id.go:140] ["IDAllocator pickCanDoFunc"] [need=1] [total=199913] [remainReqCnt=0]
2022-06-15T08:52:22.982843274Z [2022/06/15 08:52:22.982 +00:00] [DEBUG] [impl.go:384] ["HasCollection enqueued"] [traceID=70a66fa9d5ef1268] [role=proxy] [MsgID=433922062977335384] [BeginTS=433922544221290498] [EndTS=433922544221290498] [db=] [collection=banban_image_machine_database]
2022-06-15T08:52:22.983040383Z [2022/06/15 08:52:22.982 +00:00] [DEBUG] [root_coord.go:1354] [HasCollection] [role=rootcoord] ["collection name"=banban_image_machine_database] [msgID=433922062977335384]
2022-06-15T08:52:22.983076678Z [2022/06/15 08:52:22.983 +00:00] [DEBUG] [root_coord.go:1373] ["HasCollection success"] [role=rootcoord] ["collection name"=banban_image_machine_database] [msgID=433922062977335384]
2022-06-15T08:52:22.983200183Z [2022/06/15 08:52:22.983 +00:00] [DEBUG] [impl.go:414] ["HasCollection done"] [traceID=70a66fa9d5ef1268] [role=proxy] [MsgID=433922062977335384] [BeginTS=433922544221290498] [EndTS=433922544221290498] [db=] [collection=banban_image_machine_database]
2022-06-15T08:52:22.985508148Z [2022/06/15 08:52:22.985 +00:00] [DEBUG] [impl.go:632] ["DescribeCollection received"] [traceID=622579d152ae7967] [role=proxy] [db=] [collection=banban_image_machine_database]
2022-06-15T08:52:22.985716055Z [2022/06/15 08:52:22.985 +00:00] [DEBUG] [id.go:140] ["IDAllocator pickCanDoFunc"] [need=1] [total=199912] [remainReqCnt=0]
2022-06-15T08:52:22.985732500Z [2022/06/15 08:52:22.985 +00:00] [DEBUG] [impl.go:656] ["DescribeCollection enqueued"] [traceID=622579d152ae7967] [role=proxy] [MsgID=433922062977335385] [BeginTS=433922544234397697] [EndTS=433922544234397697] [db=] [collection=banban_image_machine_database]
2022-06-15T08:52:22.985903037Z [2022/06/15 08:52:22.985 +00:00] [DEBUG] [root_coord.go:1394] [DescribeCollection] [role=rootcoord] ["collection name"=banban_image_machine_database] [msgID=433922062977335385]
2022-06-15T08:52:22.985953067Z [2022/06/15 08:52:22.985 +00:00] [DEBUG] [root_coord.go:1412] ["DescribeCollection success"] [role=rootcoord] ["collection name"=banban_image_machine_database] [msgID=433922062977335385]
2022-06-15T08:52:22.986108190Z [2022/06/15 08:52:22.986 +00:00] [DEBUG] [impl.go:689] ["DescribeCollection done"] [traceID=622579d152ae7967] [role=proxy] [MsgID=433922062977335385] [BeginTS=433922544234397697] [EndTS=433922544234397697] [db=] [collection=banban_image_machine_database]
2022-06-15T08:52:22.988589435Z [2022/06/15 08:52:22.988 +00:00] [DEBUG] [impl.go:2413] ["Search received"] [traceID=10777e96486bce] [role=proxy] [db=] [collection=banban_image_machine_database] [partitions="[]"] [dsl=] [len(PlaceholderGroup)=42] [OutputFields="[]"] [search_params="[{\"key\":\"anns_field\",\"value\":\"hash_vector\"},{\"key\":\"topk\",\"value\":\"1\"},{\"key\":\"metric_type\",\"value\":\"HAMMING\"},{\"key\":\"params\",\"value\":\"{}\"},{\"key\":\"round_decimal\",\"value\":\"-1\"}]"] [travel_timestamp=0] [guarantee_timestamp=433922542924464128]
2022-06-15T08:52:22.988774678Z [2022/06/15 08:52:22.988 +00:00] [DEBUG] [id.go:140] ["IDAllocator pickCanDoFunc"] [need=1] [total=199911] [remainReqCnt=0]
2022-06-15T08:52:22.988796417Z [2022/06/15 08:52:22.988 +00:00] [DEBUG] [impl.go:2454] ["Search enqueued"] [traceID=10777e96486bce] [role=proxy] [msgID=433922062977335386] [timestamp=433922544234397698] [db=] [collection=banban_image_machine_database] [partitions="[]"] [dsl=] [len(PlaceholderGroup)=42] [OutputFields="[]"] [search_params="[{\"key\":\"anns_field\",\"value\":\"hash_vector\"},{\"key\":\"topk\",\"value\":\"1\"},{\"key\":\"metric_type\",\"value\":\"HAMMING\"},{\"key\":\"params\",\"value\":\"{}\"},{\"key\":\"round_decimal\",\"value\":\"-1\"}]"] [travel_timestamp=0] [guarantee_timestamp=433922542924464128]
2022-06-15T08:52:22.988997516Z [2022/06/15 08:52:22.988 +00:00] [DEBUG] [impl.go:93] ["show collection start"] [role=querycoord] [collectionIDs="[]"] [msgID=433922062977335386]
2022-06-15T08:52:22.989049800Z [2022/06/15 08:52:22.989 +00:00] [DEBUG] [impl.go:121] ["show collection end"] [role=querycoord] [collections="[433719451781431297]"] [inMemoryPercentage="[100]"] [msgID=433922062977335386]
2022-06-15T08:52:22.989160891Z [2022/06/15 08:52:22.989 +00:00] [DEBUG] [task.go:1484] ["QueryCoord show collections"] [collID=433719451781431297] [collections="[433719451781431297]"]
2022-06-15T08:52:22.989173320Z [2022/06/15 08:52:22.989 +00:00] [DEBUG] [task.go:1510] ["translate output fields"] [OutputFields="[]"]
2022-06-15T08:52:22.989202386Z [2022/06/15 08:52:22.989 +00:00] [DEBUG] [task.go:1557] ["create query plan"] [dsl=] ["anns field"=hash_vector] ["query info"="topk:1 metric_type:\"HAMMING\" search_params:\"{}\" round_decimal:-1 "]
2022-06-15T08:52:22.989240995Z [2022/06/15 08:52:22.989 +00:00] [DEBUG] [task.go:1599] [Proxy::searchTask::PreExecute] [plan.OutputFieldIds="[]"] [plan="vector_anns:<is_binary:true field_id:101 query_info:<topk:1 metric_type:\"HAMMING\" search_params:\"{}\" round_decimal:-1 > placeholder_tag:\"$0\" > "]
2022-06-15T08:52:22.989303009Z [2022/06/15 08:52:22.989 +00:00] [DEBUG] [time_recorder.go:78] ["proxy execute search 433922062977335386: get used message stream (0ms)"]
2022-06-15T08:52:22.989431496Z [2022/06/15 08:52:22.989 +00:00] [DEBUG] [time_recorder.go:78] ["search: send message done (0ms)"]
2022-06-15T08:52:22.989438455Z [2022/06/15 08:52:22.989 +00:00] [DEBUG] [task.go:1721] ["proxy sent one searchMsg"] [collectionID=433719451781431297] [msgID=433922062977335386] ["length of search msg"=1] [timeoutTs=0]
2022-06-15T08:52:22.989441994Z [2022/06/15 08:52:22.989 +00:00] [DEBUG] [time_recorder.go:78] ["proxy execute search 433922062977335386: send search msg to message stream (0ms)"]
2022-06-15T08:52:22.989446155Z [2022/06/15 08:52:22.989 +00:00] [DEBUG] [time_recorder.go:78] ["proxy execute search 433922062977335386: done (0ms)"]
2022-06-15T08:52:22.989624412Z [2022/06/15 08:52:22.989 +00:00] [DEBUG] [query_collection.go:350] ["check if query timeout"] [collectionID=433719451781431297] [msgID=433922062977335386] [TimeoutTs=0] [curTime=433922544235708416] [timeoutTsPhysical=1970/01/01 00:00:00.000 +00:00] [curTimePhysical=2022/06/15 08:52:22.989 +00:00]
2022-06-15T08:52:22.989634484Z [2022/06/15 08:52:22.989 +00:00] [DEBUG] [time_recorder.go:78] ["receiveQueryMsg 433922062977335386: get searchable time done (0ms)"]
2022-06-15T08:52:22.989646100Z [2022/06/15 08:52:22.989 +00:00] [DEBUG] [query_collection.go:557] ["doing query in receiveQueryMsg..."] [collectionID=433719451781431297] [sm.GuaranteeTimestamp=2022/06/15 08:52:17.987 +00:00] [serviceTime=2022/06/15 08:52:22.734 +00:00] ["delta seconds"=18446744072] [msgID=433922062977335386] [msgType=search]
2022-06-15T08:52:22.989723816Z [2022/06/15 08:52:22.989 +00:00] [DEBUG] [query_collection.go:1082] ["historical search start"] [msgID=433922062977335386]
2022-06-15T08:52:22.989769050Z [2022/06/15 08:52:22.989 +00:00] [DEBUG] [historical.go:124] ["no partition specified, search all partitions"] [collectionID=433719451781431297] ["all partitions"="
2022-06-15T08:52:22.989862220Z [2022/06/15 08:52:22.989 +00:00] [DEBUG] [streaming.go:163] ["get segmentIDs by vChannel"] [collectionID=433719451781431297] [vChannel=by-dev-rootcoord-dml_7_433719451781431297v1] [partitionID=433719451781431298] [segmentIDs="[]"]
g.go:163] ["get segmentIDs by vChannel"] [collectionID=433719451781431297] [vChannel=by-dev-rootcoord-dml_6_433719451781431297v0] [partitionID=433771218772688897] [segmentIDs="[]"]
2022-06-15T08:52:22.993062871Z [2022/06/15 08:52:22.993 +00:00] [DEBUG] [streaming.go:163] ["get segmentIDs by vChannel"] [collectionID=433719451781431297] [vChannel=by-dev-rootcoord-dml_6_433719451781431297v0] [partitionID=433771730136989697] [segmentIDs="[]"]
[partitionID=433747720748400641] [segmentIDs="[433747722662313986]"]
2022-06-15T08:52:22.994359924Z [2022/06/15 08:52:22.994 +00:00] [DEBUG] [segment.go:313] ["do search on segment"] [segmentID=433747722662313986] [segmentType=2]
2022-06-15T08:52:22.994502027Z [2022/06/15 08:52:22.994 +00:00] [DEBUG] [streaming.go:163] ["get segmentIDs by vChannel"] [collectionID=433719451781431297] [vChannel=by-dev-rootcoord-dml_6_433719451781431297v0] [partitionID=433770752549847041] [segmentIDs="[433770753598160897]"]
2022-06-15T08:52:22.994524004Z [2022/06/15 08:52:22.994 +00:00] [DEBUG] [segment.go:313] ["do search on segment"] [segmentID=433770753598160897] [segmentType=2]
2022-06-15T08:52:22.994701365Z [2022/06/15 08:52:22.994 +00:00] [DEBUG] [query_collection.go:1098] ["streaming search"] [msgID=433922062977335386] [collectionID=433719451781431297] ["searched dmChannel"=by-dev-rootcoord-dml_6_433719451781431297v0] ["searched partitionIDs"="[433719451781431298,433719613341040641,433720147066224641,433720311915216897,433721078188605441,433722417285431297,433722891543248897,433723278625079297,433723675707965441,433724113999626241,433724505341296641,433724904717680641,433725862460522497,433726423619338241,433726638747811841,433727090487197697,433727502014218241,433727914039050241,433728356367728641,433729007284387841,433729641227223041,433730226319523841,433730689069219841,433731150049181697,433731640769904641,433732219020247041,433732745025290241,433733184706052097,433733586245386241,433734014169251841,433734540291997697,433735092026736641,433735496541143041,433736087112253441,433736723401015297,433737264387588097,433737672873738241,433738177841463297,433738615962992641,433739070324080641,433739599723626497,433740116737392641,433740611638788097,433741137840701441,433741613383024641,433742109162602497,433742656650346497,433743253722628097,433743886459863041,433744460686295041,433744996482154497,433745514990141441,433746015069143041,433746541480509441,433747097513885697,433748320861552641,433748986405847041,433749590411837441,433750183302660097,433750813405347841,433751482147799041,433752114832343041,433752656238084097,433753294375485441,433753874696765441,433754434688778241,433755046034800641,433755555930832897,433756125098147841,433756757271511041,433757293159383041,433757971928842241,433758438754615297,433758989637386241,433759307434557441,433759977671229441,433760504239882241,433761136163946497,433761663951568897,433762133399044097,433762611300663297,433763369932292097,433763953832099841,433764635222736897,433765136088432641,433765555846250497,433766037339504641,433766506826301441,433767048533770241,433767547249623041,433768201967370241,433768642893316097,433769089835728897,433769476616093697,433769922837872641,433770277400477697,433771218772688897,433771730136989697,433772215352688641,433772741095587841,433773258030186497,433773649122820097,433773748462551041,433773832453488641,433774440928772097,433775043663626241,433775627051991041,433776191592202241,433776786514903041,433776872143978497,433777580273827841,433778211778723841,433778825431613441,433779403786813441,433780117447639041,433780754890096641,433780996009885697,433781624539709441,433720787693731841,433720909905264641,433721477892669441,433721875381354497,433725385515991041,433747720748400641,433770752549847041]"] ["searched segmentIDs"="[433922081052164097,433720789161738241,433720911215984641,433721479465533442,433721876810301442,433725386944413698,433747722662313986,433770753598160897]"]
2022-06-15T08:52:22.994719843Z [2022/06/15 08:52:22.994 +00:00] [DEBUG] [time_recorder.go:78] ["search 433719451781431297(nq=1, k=1), msgID = 433922062977335386: streaming search done, msgID = 433922062977335386 (4ms)"]
2022-06-15T08:52:22.994724222Z [2022/06/15 08:52:22.994 +00:00] [DEBUG] [query_collection.go:1149] ["QueryNode reduce data"] [msgID=433922062977335386] [numSegment=15]
2022-06-15T08:52:22.994828011Z 2022-06-15 08:52:22,994 DEBUG [default] [user@0a0c66ef9d32] [void ReduceResultData(std::vector<milvus::SearchResult*>&, int64_t, int64_t)] [/go/src/github.com/milvus-io/milvus/internal/core/src/segcore/reduce_c.cpp:131] [[[SEGCORE](https://github.com/milvus-io/milvus/issues#)](https://github.com/milvus-io/milvus/issues#)][ReduceResultData][milvus] skip duplicated search result, count = 0
2022-06-15T08:52:22.994864551Z [2022/06/15 08:52:22.994 +00:00] [DEBUG] [query_collection.go:1152] ["QueryNode reduce data finished"] [msgID=433922062977335386]
2022-06-15T08:52:22.995147762Z [2022/06/15 08:52:22.995 +00:00] [DEBUG] [query_collection.go:718] [translateHits:] [lenOfFieldIDs=0] [lenOfRawHits=1]
2022-06-15T08:52:22.995179858Z [2022/06/15 08:52:22.995 +00:00] [DEBUG] [query_collection.go:1228] ["QueryNode SearchResultMsg"] [collectionID=433719451781431297] [msgID=433922062977335386] [vChannels="[by-dev-rootcoord-dml_7_433719451781431297v1,by-dev-rootcoord-dml_6_433719451781431297v0]"] [sealedSegmentSearched="[]"]
2022-06-15T08:52:22.995408752Z [2022/06/15 08:52:22.995 +00:00] [DEBUG] [task_scheduler.go:681] ["got a search result"] [role=proxy] [ReqID=433922062977335386]
2022-06-15T08:52:22.995416970Z [2022/06/15 08:52:22.995 +00:00] [DEBUG] [task_scheduler.go:701] ["first receive search result of this task"] [role=proxy] [reqID=433922062977335386]
2022-06-15T08:52:22.995424756Z [2022/06/15 08:52:22.995 +00:00] [DEBUG] [task_scheduler.go:717] ["process search result"] [role=proxy] [collection=banban_image_machine_database] [reqID=433922062977335386] ["answer cnt"=1]
2022-06-15T08:52:22.995440541Z [2022/06/15 08:52:22.995 +00:00] [DEBUG] [task_scheduler.go:609] ["check if result buf is ready to reduce"] [role=proxy] [MsgID=433922062977335386] [receivedVChansSet="[\"by-dev-rootcoord-dml_7_433719451781431297v1\",\"by-dev-rootcoord-dml_6_433719451781431297v0\"]"] [usedVChans="[\"by-dev-rootcoord-dml_6_433719451781431297v0\",\"by-dev-rootcoord-dml_7_433719451781431297v1\"]"] [receivedSealedSegmentIDsSet="[]"] [receivedGlobalSegmentIDsSet="[433719455097815041]"]
2022-06-15T08:52:22.995471959Z [2022/06/15 08:52:22.995 +00:00] [DEBUG] [proxy_session_manager.go:108] ["success to send search result"] [node=16] [base="msg_type:SearchResult msgID:433922062977335386 timestamp:433922544234397698 sourceID:16 "]
2022-06-15T08:52:22.995570554Z [2022/06/15 08:52:22.995 +00:00] [DEBUG] [time_recorder.go:78] ["search 433719451781431297(nq=1, k=1), msgID = 433922062977335386: publish search result, msgID = 433922062977335386 (0ms)"]
2022-06-15T08:52:22.995577895Z [2022/06/15 08:52:22.995 +00:00] [DEBUG] [time_recorder.go:78] ["search 433719451781431297(nq=1, k=1), msgID = 433922062977335386: all done, msgID = 433922062977335386 (5ms)"]
2022-06-15T08:52:22.995581207Z [2022/06/15 08:52:22.995 +00:00] [DEBUG] [time_recorder.go:78] ["receiveQueryMsg 433922062977335386: operation done (5ms)"]
2022-06-15T08:52:22.995584338Z [2022/06/15 08:52:22.995 +00:00] [DEBUG] [query_collection.go:589] ["do query done in receiveQueryMsg"] [collectionID=433719451781431297] [msgID=433922062977335386] [msgType=search]
2022-06-15T08:52:22.995587599Z [2022/06/15 08:52:22.995 +00:00] [DEBUG] [time_recorder.go:78] ["receiveQueryMsg 433922062977335386: all done (5ms)"]

**Comment by yanliang567** [comment]
@chuzhifeng  sounds like a known issue which occasionally reproduces in 2.0.2. Could you please retry with latest master build, we believe it is fixed now.

/assign @chuzhifeng 
/unassign

**Comment by chuzhifeng** [comment]
我是通过docker-compose启动服务milvus的，
<img width="1445" alt="image" src="https://user-images.githubusercontent.com/32004153/174015850-dcb5fa5f-75bc-45ee-b475-92a32e153f5b.png">
看你们[Releases](https://github.com/milvus-io/milvus/releases)的最终版本就是v2.0.2啊

**Comment by soothing-rain** [comment]
> 或者有最新版的[milvus-standalone-docker-compose.yml](https://github.com/milvus-io/milvus/releases/download/v2.0.2/milvus-standalone-docker-compose.yml)下载链接吗，我并没有找到最新版的docker-compose配置文件

Hey 2.1.0 is not officially released. Please expect the first candidate version within a week or so :)

**Comment by chang48** [comment]
I'm having a similar issue even with the latest dev container image.

**Comment by yanliang567** [comment]
> I'm having a similar issue even with the latest dev container image.

@chang48 what about trying to release and reload the collection ?

**Comment by chang48** [comment]
> > I'm having a similar issue even with the latest dev container image.
> 
> @chang48 what about trying to release and reload the collection ?

@yanliang567 Turns out it was not a milvus issue. I was building a semantic search engine using haystack's milvus backend. The problem was when the service was restarted, I couldn't find any indexed data even though the folder `volume` inside the container was not empty. After talking to the haystack team, I realized the indexed data is stored in haystack's own DB backend. By setting the DB parameters properly, I was able to reconnect the service and retrieved the indexed data, as well as the embedded vectors.
